pepper/Makefile

```diff
@@ -43,14 +44,19 @@ link-exo: check-env
 	ln -sf ~/dex-zksnarks/exo_compute/private_input.py $(PEPPER_BIN)/exo0
 
 # Target to run all tests (e.g. used by travis)
-test: unit-tests gadget-tests e2e-tests
+test: $(FILENAMETESTS)%o gadget-tests e2e-tests
```

Comment:
nit: maybe make this a sub-target called unit-tests or app-tests, so it's clear that this is only for those types of files.